editor/src/components/inspector/controls/color-picker.tsx

```diff
@@ -716,8 +715,9 @@ export class ColorPickerInner extends React.Component<
               minimum={0}
               maximum={1}
               stepSize={0.01}
-              DEPRECATED_labelBelow='S'
+              scrubbable_innerlabel='S'
```

Comment:
If we're removing the `DEPRECATED_` prefix, please use camelCase
```suggestion
              scrubbableInnerLabel='S'
```

Reply:
can do!